### ISSUE TITLE
NTR: Cortical dysplasia

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#4057]
Synonyms: HP:0002539 
; HPO ID: HP:0007139 
; Alternative ID: Abnormality of the cerebral cortex 
; Son of_1: NA 
; Son of_2: NA 
; Son of_3: NA 
; Son of_4: The presence of developmental dysplasia of the cerebral cortex. 
; Textual definition: NA 
; PMID to support changes: COST:neuromig 
; label: NA 
; Curator: n/a 
; OMIM -disease: n/a 
; OMIM -gene

**Comment by pnrobinson** [comment]
This term already exists
https://hpo.jax.org/app/browse/term/HP:0002539
